docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+
+  localstack:
```

Comment:
No setup script? Rather you follow https://github.com/UKHomeOffice/hocs-audit/commit/ab4c3664814f83d5929e05cd88fe03c793c58eb7 as this way the queues will only be created once. Will allow you to create a DLQ for it too, to stop message pollution on the main Queue for failed messaged...